packages/component/src/BasicTranscript.tsx

```diff
@@ -602,13 +612,15 @@ InternalTranscript.propTypes = {
 };
 
 type InternalTranscriptScrollableProps = {
+  ariaBusy: boolean;
```

Comment:
```suggestion
  'aria-busy': boolean;
```

Reply:
Fixed, though it took a little more rearranging to make the linter happy.